src/hotspot/share/gc/g1/g1RemSet.cpp

```diff
@@ -244,12 +239,12 @@ class G1RemSetScanState : public CHeapObj<mtGC> {
 #endif
     }
 
-    static uint chunk_size() { return M; }
-
     void do_work(uint worker_id) override {
+      const uint num_regions_per_worker = num_cards_per_worker / (uint) HeapRegion::CardsPerRegion;
```

Comment:
```suggestion
      const uint num_regions_per_worker = num_cards_per_worker / (uint)HeapRegion::CardsPerRegion;
```
Superfluous whitespace.